plugin/Controller/EstateTitleBuilder.php

```diff
@@ -109,6 +110,44 @@ public function buildTitle(int $estateId, string $format): string
 		return '';
 	}
 
+	/**
+	 *
+	 * @param int $estateId
+	 * @param string $format
+	 *
+	 * The Format consists of:
+	 * %1$s: 'objekttitel',
+	 * %2$s: 'objektbeschreibung',
+	 * %3$s: 'ort',
+	 * %4$s: 'plz',
+	 * %5$s: 'objektart',
+	 * %6$s: 'vermarktungsart',
+	 * %7$s: 'Id'
+	 *
+	 * @return string
+	 * @throws Exception
+	 */
+
+	public function buildCustomFieldTitle(int $estateId, string $format): string
+	{
+		$this->_pDefaultFilterBuilder->setEstateId($estateId);
+		$this->_pEstateDetail->loadSingleEstate($estateId);
```

Comment:
Does this mean that we send two requests to enterprise?

Reply:
Yes, I will send two requests

Reply:
Ok, I'll have to check the performance. The double request will only be sent if the option to set custom fields is active, correct?